collada_urdf/package.xml

```diff
@@ -25,6 +25,7 @@
   <build_depend>collada_parser</build_depend>
   <build_depend>liburdfdom-dev</build_depend>
   <build_depend>liburdfdom-headers-dev</build_depend>
+  <build_depend>pcrecpp</build_depend>
```

Comment:
Is `pcrecpp` different from `pcre` or `pcre-dev`?

https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L2619-L2632

I don't see `pcrecpp` in the `rosdep` database.